src/components/pastAnalysis/PastAnalysisTable.tsx

```diff
@@ -128,9 +128,10 @@ const PastAnalysisTable: React.FC = () => {
   ]
   const [rows, setRows] = useState<(tableRowsChild | tableRowsParent)[]>([])
 
-  // Stores an array of the "Analysis Created" property of the rows of page 0 of the table
-  // Used to identify which rows are new and need to be highlighted green
-  const newRowsRef = useRef<string[]>([]);
+  // Stores an array of either empty arrays or arrays with the feedIds for each row
+  const currRowsFeedIdsRef = useRef<number[][]>([]);
+  // Stores a list of the feeds that have just finished
+  const finishedFeedsRef = useRef<Set<number>>(new Set());
```

Comment:
Can we revert back to using array of strings rather than `Set` so that all browsers are compatible.